src/libNetwork/Guard.cpp

```diff
@@ -213,11 +213,23 @@ void Guard::AddToExclusionList(const uint128_t& ft, const uint128_t& sd) {
   }
 }
 
+void Guard::ValidateRunTimeEnvironment() {
+  unsigned int nodeReplacementLimit = COMM_SIZE - (COMM_SIZE - (COMM_SIZE / 3));
```

Comment:
Why not just use nodeReplacementLimit = COMM_SIZE / 3, the result is the same.

Reply:
Miror the logic from the re balancing side. Not sure whether replacing with `COMM_SIZE / 3` will have a corner case or not. What do you think?